### PR TITLE
Add file extensions

### PR DESCRIPTION
**Pull request by smithjp**
https://jira.lib.unc.edu/browse/HYC-492

### CODE REVIEW

File: lib/tasks/migrate/services/ingest_service.rb [hunk 1]
```diff
@@ -160,13 +172,26 @@ def ingest_records
...
+        else
+          extension = nil
+          puts "[#{Time.now.to_s}] #{parent.id} expected mime_type: #{resource[:mime_type]}"
+        end
+
+        # Update file title
+        if resource['title'].first.match?(/.*#{extension}\z/)
+          extension = ''
+        end
+        resource['title'] = resource['title'].map{|title| title.split(' ').join('_')+extension}
```
Comment (bbpennel) [inline lib/tasks/migrate/services/ingest_service.rb]: I'd like to double check that people are okay with us changing the stored file title, versus adding the extension in on retrieval

File: lib/tasks/migration/mime_types.csv [hunk 2]
```diff
@@ -0,0 +1,150 @@
+type,extension
```
Comment (bbpennel) [inline lib/tasks/migration/mime_types.csv]: Given the addition of mimetype.yml, is this file still needed?

Comment (smithjp) [reply lib/tasks/migration/mime_types.csv]: no, I will remove it
